client/Cargo.lock

```diff
@@ -4894,24 +4894,24 @@ checksum = "9c8d87e72b64a3b4db28d11ce29237c246188f4f51057d65a7eab63b7987e423"
 
 [[package]]
 name = "wasm-bindgen"
-version = "0.2.95"
+version = "0.2.100"
```

Comment:
This change brings `wasm-bindgen` out of alignment with the `flake.lock`, so the build script will reset it.